src/AWS/Orleans.Streaming.SQS/Streams/SqsStreamOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace Orleans.Configuration
+{
+    public class SqsStreamOptions : PersistentStreamOptions
+    {
+        public string ClusterId { get; set; }
+
+        [Redact]
+        public string ConnectionString { get; set; }
```

Comment:
Should this be `[RedactConnectionString]`? Xiao made a change to redact the entire thing by default if no redactions were made.

BTW, @xiazen, that change also means that `UseDevelopmentStorage=true` would be redacted, but better to be safe than sorry.

Reply:
I don't think so.  I'm unconvinced RedactConnectionString is safe for unknown formats.  By the logic of the call, if any of the patterns match the string is logged with the recognized patterns redacted.  This means if a connection string has a matched, and an unmatched secret, the unmatched secret will logged. which is not acceptable.

Reply:
ok, no worries. In the future we can create a version which only includes known-good portions of the connection string, instead. We can wait until someone complains, if they ever do.